testData/highlighting/unitializedStructInitialization.go

```diff
@@ -0,0 +1,30 @@
+package demo2094
+
+type demo struct {
+	a string
+	b int32
+	c string
+}
+
+type bemo struct {
+	demo
+	x string
+}
+
+func _() (struct {
+	x string
+}) {
+	_ = struct{ x string }{x: "demo"}
+	_ = <weak_warning descr="Structure has unnamed field initialization">demo{"demo"}</weak_warning>
+	b, _ := <weak_warning descr="Structure has unnamed field initialization">demo{"demo"}</weak_warning>, 1
+	_ = <weak_warning descr="Structure has unnamed field initialization">demo{"demo", 1, "demo"}</weak_warning>
+	_ = demo{a: "demo"}
+	_ = demo{a: "demo", b: 1}
+	_ = <weak_warning descr="Structure has unnamed field initialization">demo{
```

Comment:
> I believe it's not passed now. 

Not sure what it means, the tests passed on last run.

> Also it's not structure has field anymore, it's unnamed field

Actually that case should be an error: `mixture of field:value and value initializers` now that I think of it, see: https://play.golang.org/p/qZpTKwV6q0 

Should I introduce a different inspection for that and abort this inspection if it would match for the other case?